ccmlib/node.py

```diff
@@ -1418,11 +1539,69 @@ def drain(self, block_on_log=False):
         if block_on_log:
             self.watch_log_for("DRAINED", from_mark=mark)
 
-    def repair(self, options=[], **kwargs):
-        args = ["repair"] + options
-        cmd = ' '.join(args)
+    @property
+    def scylla_yaml(self) -> Dict[str, Any]:
+        return YAML().load(Path(self.get_conf_dir()) / common.SCYLLA_CONF)
+
+    @property
+    def api_port(self) -> int:
+        return self.scylla_yaml.get('api_port', 10000)
+
+    def cluster_command_available(self):
+        stdout, _ = self.nodetool('help')
+        return "  cluster  " in stdout
+
+    def run_repair(self, options, **kwargs):
+        cmd = "repair " + options
+        return self.nodetool(cmd, **kwargs)
+
+    def run_cluster_repair(self, options, **kwargs):
+        cmd = "cluster repair " + options
         return self.nodetool(cmd, **kwargs)
 
+    def repair(self, options=[], replication: RepairOptions.KeyspaceReplication = RepairOptions.KeyspaceReplication.ALL, keyspace: Optional[str] = None,
+               tables: Optional[list[str]] = None, hosts: Optional[list[str]] = None, dcs: Optional[list[str]] = None, dc_parallel: bool = False,
+               end_token: Optional[int] = None, full: bool = False, local: bool = False, ignore_unreplicated_keyspaces: bool = False,
+               job_threads: Optional[int] = None, partitioner_range: bool = False, pull: bool = False, sequential: bool = False, start_token: Optional[int] = None,
+               trace: bool = False, tablet_tokens: Optional[list[str]] = None, **kwargs):
+        if not options:
```

Comment:
Let's drop the extra options that might be passed by kwargs as raw repair options.

Reply:
I'll do it as a follow-up so that dtest does not fail after this change (and before its)